.github/workflows/test-all.yml

```diff
@@ -84,6 +84,336 @@ jobs:
+          path: /tmp/hermes/hermes-runtime-darwin
+      - name: Upload osx-bin
+        uses: actions/upload-artifact@v4.3.1
+        with:
+          name: hermes-osx-bin-${{ matrix.flavor }}
+          path: /tmp/hermes/osx-bin
+      - name: Create dSYM archive
+        if: ${{ steps.check_if_apple_artifacts_are_there.outputs.ARTIFACTS_EXIST != true }}
+        run: |
+          FLAVOR=${{ matrix.flavor }}
+          WORKING_DIR="/tmp/hermes_tmp/dSYM/$FLAVOR"
+
+          mkdir -p "$WORKING_DIR/macosx"
+          mkdir -p "$WORKING_DIR/catalyst"
+          mkdir -p "$WORKING_DIR/iphoneos"
+          mkdir -p "$WORKING_DIR/iphonesimulator"
+
+          cd ./packages/react-native/sdks/hermes || exit 1
+
+          DSYM_FILE_PATH=API/hermes/hermes.framework.dSYM
+          cp -r build_macosx/$DSYM_FILE_PATH "$WORKING_DIR/macosx/"
+          cp -r build_catalyst/$DSYM_FILE_PATH "$WORKING_DIR/catalyst/"
+          cp -r build_iphoneos/$DSYM_FILE_PATH "$WORKING_DIR/iphoneos/"
+          cp -r build_iphonesimulator/$DSYM_FILE_PATH "$WORKING_DIR/iphonesimulator/"
+
+          DEST_DIR="/tmp/hermes/dSYM/$FLAVOR"
+          tar -C "$WORKING_DIR" -czvf "hermes.framework.dSYM" .
+
+          mkdir -p "$DEST_DIR"
+          mv "hermes.framework.dSYM" "$DEST_DIR"
+      - name: Upload hermes dSYM acrifacts
+        uses: actions/upload-artifact@v4.3.1
+        with:
+          name: hermes-dSYM-${{ matrix.flavor }}
+          path: /tmp/hermes/dSYM/${{ matrix.flavor }}
+  test_ios_rntester_ruby_3_2_0:
+    runs-on: macos-13
+    needs: [build_apple_slices_hermes, prepare_hermes_workspace, build_hermes_macos]
+    env:
+      HERMES_WS_DIR: /tmp/hermes
+      HERMES_TARBALL_ARTIFACTS_DIR: /tmp/hermes/hermes-runtime-darwin
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      - name: Run it
+        uses: ./.github/actions/test_ios_rntester
+        with:
+          ruby-version: "3.2.0"
+          hermes-version: ${{ needs.prepare_hermes_workspace.outputs.hermes-version }}
+          react-native-version: ${{ needs.prepare_hermes_workspace.outputs.react-native-version }}
+  test_ios_rntester1:
+    runs-on: macos-13
+    needs: [build_apple_slices_hermes, prepare_hermes_workspace, build_hermes_macos]
+    env:
+      HERMES_WS_DIR: /tmp/hermes
+      HERMES_TARBALL_ARTIFACTS_DIR: /tmp/hermes/hermes-runtime-darwin
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        jsengine: [Hermes, JSC]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      - name: Run it
+        uses: ./.github/actions/test_ios_rntester
+        with:
+          jsengine: ${{ matrix.jsengine }}
+          use-frameworks: DynamicFrameworks
+          hermes-version: ${{ needs.prepare_hermes_workspace.outputs.hermes-version }}
+          react-native-version: ${{ needs.prepare_hermes_workspace.outputs.react-native-version }}
+  test_ios_rntester2:
```

Comment:
this can probably be just `test_ios_rntester`. I don't see any workflow with that name now. Am I missing something?

Reply:
Done.